eksctl/cluster-config.yaml

```diff
@@ -6,9 +6,18 @@ metadata:
   region: us-west-2
 
 managedNodeGroups:
-  - name: eks-mng
+  - name: helper1
+    labels: { releaseName: h1 }
     instanceType: t3.medium
-    desiredCapacity: 2
+    desiredCapacity: 1
+  - name: helper2
+    labels: { releaseName: h2 }
+    instanceType: t3.medium
+    desiredCapacity: 1
+  - name: helper3
+    labels: { releaseName: h3 }
+    instanceType: t3.medium
```

Comment:
we aren't utilizing our current boxes completely, but we may want larger boxes than t3.medium. I think we want compute optimized, so maybe `c7i.large` or `c7i.xlarge`?

Reply:
Can I do that on a future change? I'm not quite sure what we need to use.